pkg/agent/qrm-plugins/cpu/dynamicpolicy/state/state.go

```diff
@@ -402,6 +407,7 @@ func (ns *NUMANodeState) SetAllocationInfo(podUID string, containerName string,
 }
 
 // GetDefaultCPUSet returns default cpuset in this node
+// TODO: add lock to NUMANodeMap
```

Comment:
since stateCheckpoint functions are all under locks, it seems that we don't need to add lock in the operations in 
 specifc state struct.

Reply:
removed.